### ISSUE TITLE
Implementation questions.

### ISSUE THREAD

**Issue by jonalf** [swang01/final#1]
0) Whys is this in the period 5 directory?

The idea is approved, but I have questions about your usage of signals and multiple processes. If the game is between multiple computers, then signals will not work, they are only meant for process communication on one computer. Also, coordinating the child processes to "calculate statistics" may be unnecessarily complicated, but I can be convinced it is what you want to do, just provide more details.

**Comment by jonalf** [comment]
0) period 5 makes sense, update your README to reflect Vivian's actual class period.

**Comment by swang01** [comment]
We will use forking servers to allow for signal communication between a subserver and client processes.  We also decided not to calculate statistics on child processes but rather on the parent process by simply storing stats in variables. The readme has been updated accordingly

**Comment by jonalf** [comment]
Signals are ways for processes on the same computer to communicate. They cannot be sent across pipes. What is the "parent process" you keep referring to. Is this the main server process?

**Comment by RLL24187** [comment]
I updated the README to clarify that our parent process is the main server process and the subservers will serve as the child processes. We also decided that it would be easier to have subservers calculate the statistics because this would avoid a problem with the main server and needing to sort information from each client to run the calculation functions for each, whereas the subservers would only be paired with a single client and this issue can be avoided. We'll use pipes to provide communication between the different processes (subservers and main server) on different computers across the network and thus transfer information about the statistics and positions to each client.
